test/helpers/utils.go

```diff
@@ -811,6 +813,72 @@ func ExecCmdOnNode(t *testing.T, cs *framework.ClientSet, node corev1.Node, subA
 	return string(out)
 }
 
+// Gets the rpm-ostree status for a given node.
+func GetRPMOStreeStatusForNode(t *testing.T, cs *framework.ClientSet, node corev1.Node) *rpmostreeclient.Status {
+	status, err := getRPMOStreeStatusForNode(cs, node)
+	require.NoError(t, err)
+
+	return status
+}
+
+// Internal-only version of GetRPMOStreeStatusForNode()
+func getRPMOStreeStatusForNode(cs *framework.ClientSet, node corev1.Node) (*rpmostreeclient.Status, error) {
+	cmd, err := execCmdOnNode(cs, node, "chroot", "/rootfs", "rpm-ostree", "status", "--json")
+	if err != nil {
+		return nil, fmt.Errorf("could not construct command: %w", err)
+	}
+
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		return nil, fmt.Errorf("could not execute command %s on node %s: %w", cmd.String(), node.Name, err)
+	}
+
+	status := &rpmostreeclient.Status{}
+
+	if err := json.Unmarshal(output, status); err != nil {
+		return nil, fmt.Errorf("could not parse rpm-ostree status for node %s: %w", node.Name, err)
+	}
+
+	return status, nil
+}
+
+// Gets the currently booted rpmostree deployment for a given node.
+func getBootedRPMOstreeDeployment(cs *framework.ClientSet, node corev1.Node) (*rpmostreeclient.Deployment, error) {
+	status, err := getRPMOStreeStatusForNode(cs, node)
+	if err != nil {
+		return nil, fmt.Errorf("could not get rpm-ostree status for node %s: %w", node.Name, err)
+	}
+
+	for _, deployment := range status.Deployments {
+		deployment := deployment
+		if deployment.Booted {
+			return &deployment, nil
+		}
+	}
+
+	return nil, fmt.Errorf("no booted deployments found on node %s", node.Name)
+}
+
+// Asserts that a given node is booted into a specific image. This works by
+// examining the rpm-ostree status and determinining if the currently booted
```

Comment:
Oh also, the verify job says this and line 873 typo'ed determinining

Reply:
Thanks, will fix!